pyquil/paulis.py

```diff
@@ -282,6 +308,12 @@ def __radd__(self, other):
         assert isinstance(other, Number)
         return self + other
 
+    def __sub__(self, other):
+        return self + -1. * other
+
+    def __rsub__(self, other):
+        return other + -1. * self
+
```

Comment:
same comments as above on the `+` operator. Also why inconsistent docstrings? i.e. none here, but in the PauliTerms?